pl_bolts/models/gans/__init__.py

```diff
@@ -1,7 +1,9 @@
 from pl_bolts.models.gans.basic.basic_gan_module import GAN  # noqa: F401
 from pl_bolts.models.gans.dcgan.dcgan_module import DCGAN  # noqa: F401
+from pl_bolts.models.gans.pix2pix.pix2pix_module import Pix2Pix
```

Comment:
Not related to this PR, but could you include this suggestion as well?
```suggestion
from pl_bolts.models.gans.basic.basic_gan_module import GAN
from pl_bolts.models.gans.dcgan.dcgan_module import DCGAN
from pl_bolts.models.gans.pix2pix.pix2pix_module import Pix2Pix
```